styles.css

```
@@ -1,88 +1,87 @@
-
 body {
-    background-color: #bdc2d8;
-    color:#1c1c31;
+  background-color: #e2e5f2;
+  color: #1c1c31;
 }
 
 header {
-    margin-top:50px;
-    font-size: 2.8rem;
-    font-weight:bold;
-    text-align: center;
-    cursor:pointer;
-    user-select:none;
+  margin-top: 50px;
```

margin-top should be 60px 
```suggestion
  margin-top: 60px;
```